avaframe/ana1Tests/rotationTest.py

```diff
@@ -0,0 +1,240 @@
+    """Initialize report for rotation test
+
+    Parameters
+    ----------
+    avalancheDir : pathlib path
+        path to avalanche directory
+    dem : dict
+        dem dictionary
+    simDF : dataFrame
+        DFA simulation dataFrame
+    resTypeList : list
+        list of result types available
+    comModule : str
+        should the aimec mass analysis be done
+    refSimName : str
+        name of the reference simulation in the simDF
+
+    Returns
+    -------
+    reportRotationTest : dict
+        report dictionary
+    """
+    dateTimeInfo = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
+    reportRotationTest = {'headerLine': {'type': 'title', 'title': 'Rotation Test for DFA Simulation'},
+                          'avaName': {'type': 'avaName', 'name': str(avalancheDir)},
+                          'time': {'type': 'time', 'time': dateTimeInfo},
+                          'Simulation Parameters': {
+                          'type': 'list',
+                          'Program version': getVersion(),
+                          'DFA module': comModule,
+                          'Reference simulation': refSimName},
+                          'Rotation test input simulations': {
+                          'type': 'pandasDF', 'column names': {'simName': 'Simulation name',
+                                                               'releaseScenario': 'Release name', 'relAngle': 'Angle °'}},
+                          'Rotation test Energy line result table': {
+                          'type': 'pandasDF', 'column names': {'simName': 'Simulation name', 'relAngle': 'Angle °',
+                                                               'zDiff': 's Diff [m]',
+                                                               'sDiff': 'z Diff [m]',
+                                                               'runoutAngleDiff': 'angle Diff [°]'}},
+                          'Rotation test Energy line figures': {'type': 'image'},
+                          'Rotation test AIMEC result table': {
+                          'type': 'pandasDF', 'column names': {'simName': 'Simulation name', 'relAngle': 'Angle °',
+                                                               'sRunout': 'Runout [m]', 'relMass': 'release mass [kg]',
+                                                               'finalMass': 'final mass [kg]',
+                                                               'entMass': 'entrained mass [kg]',
+                                                               'TP': 'True positive area [-]',
+                                                               'FP': 'False positive area [-]',
+                                                               'FN': 'False negative area [-]'}},
+                          'Rotation test AIMEC figures': {'type': 'image'}}
+    resTypeList = list(set(resTypeList).intersection(['ppr', 'pfv', 'pft']))
```

Comment:
Because AIMEC can only analyze these 3. So I want to get rif of any other restype. And I am not sure all these 3 restypes are in the simDF so I remove them if not